src/components/Regions.js

```diff
@@ -304,6 +354,174 @@ const RegionTable = (props) => {
 
   return (
     <>
+      <Drawer
+        anchor="right"
+        open={drawerOpen}
+        BackdropProps={{ invisible: true }}
+        onClose={() => {
+          setDrawerOpen(false);
+        }}
+        classes={{ paper: classes.detailsDrawer }}
+      >
+        <Grid
+          container
+          direction="column"
+          className={classes.itemDrawerDetails}
+        >
+          {/* start  details header */}
+          <Grid item>
+            <Grid container direction="row" justifyContent="space-between">
+              <Grid item>
+                <Grid
+                  container
+                  direction="row"
+                  alignContent="flex-end"
+                  justifyContent="flex-start"
+                >
+                  <Typography variant="h4">Details</Typography>
+                </Grid>
+              </Grid>
+              <CloseIcon
+                onClick={() => {
+                  setDrawerOpen(false);
+                }}
+                className={classes.itemDetailsCloseIcon}
+              />
+            </Grid>
+          </Grid>
+          {/* end details header */}
+          <Grid item className={classes.itemDetailsContents}>
+            <Grid container direction="column" justifyContent="space-around">
+              <Grid item className={classes.itemRegionsDetail}>
+                <Typography>Name</Typography>
+                <Typography variant="h6">
+                  {selectedItem?.name ?? '---'}
+                </Typography>
+              </Grid>
+              {!userHasOrg && (
+                <Grid item className={classes.itemRegionsDetail}>
+                  <Typography>Owner</Typography>
+                  <Typography variant="h6">
+                    {orgList.find(
+                      (org) => org.stakeholder_uuid === selectedItem?.owner_id
+                    )?.name || '---'}
+                  </Typography>
+                </Grid>
+              )}
+              {!showCollections && (
+                <Grid item className={classes.itemRegionsDetail}>
+                  <Typography>Collection</Typography>
+                  <Typography>
+                    <Link
+                      underline="always"
+                      target="_blank"
+                      onClick={(e) => {
+                        e.stopPropagation();
+                        setShowCollections(true);
+                        let region = selectedItem;
+                        let collection = region.collection;
+                        setSelectedItem(collection);
+                      }}
+                      style={{ display: 'inline-block', cursor: 'pointer' }}
+                    >
+                      Collection details
```

Comment:
Is it possible to display the name of the collection here?

Reply:
Yes, @nmcharlton. Fixed it!

Reply:
Looks like this right now
![Screen Shot 2022-10-26 at 2 37 26 PM](https://user-images.githubusercontent.com/50210952/198143518-934c3772-18a8-4237-bee7-ff55fe02f31f.png)